client/src/offline-settings/settings.tsx

```diff
@@ -14,10 +14,22 @@ function displayEstimate({ usage = 0, quota = Infinity }: StorageEstimate) {
 }
 
 export default function SettingsApp({ ...appProps }) {
+  const serviceWorkerAvailable = window?.navigator?.serviceWorker;
+
   return (
     <section className="field-group">
       {/* <h3>Offline settings</h3> */}
-      <Settings />
+      {serviceWorkerAvailable ? (
+        <Settings />
+      ) : (
+        <>
+          <h4>Offline mode is unavailable 😭 </h4>{" "}
```

Comment:
I _really_ like emoticons personally but, is this "on brand" as they say?

Reply:
We do have them in a few places already to be honest.

Reply:
I suppose not. Will remove.